src/async_impl/client.rs

```diff
@@ -719,6 +727,23 @@ impl ClientBuilder {
         self
     }
 
+    /// Controls the use of built-in system certificates during certificate validation.
```

Comment:
```suggestion
    /// Controls the use of built-in/preloaded certificates during certificate validation.
```

It doesn't just control the system certificates, but also the ones that are hardcoded using the `webpki-roots` crate. Those are shipped with the binary and thus have nothing to do with the system certificates.